src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb

```diff
@@ -484,13 +474,21 @@ def mount_resize_fs(device, directory, fs_type, size, osize)
 
             Command.execute("#{COMMANDS[:xfs_growfs]} -d #{directory}", false)
         when /ext/
-            _rc, o, e = Command.execute("#{COMMANDS[:e2fsck]} -f -y #{device}", false)
+            err = "#{__method__}: failed to resize #{device}\n"
+
+            _rc, o, e = check_ext4(device)
```

Comment:
after poweroff they failed to boot again when recreating the fs uuid, it required e2fsck to be run again